### PR TITLE
Work and about sections

### PR DESCRIPTION
**Pull request by Marcraphael12**
This is the work section with a title, a worklist item with an image, a text, and a button.
My code is yours, so kindly leave a comment or check the issue page if you found a problem.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -136,3 +136,105 @@ body {
...
+}
+
+.work-list-item {
+  width: 100%;
+  height: auto;
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 40px;
+}
+
```
Comment (dasileker) [inline style.css:158]: - [ ] Please fix the alignment your mobile version is not aligned properly. 👉 [your project](https://user-images.githubusercontent.com/61562626/129043476-b0c6a4a2-d375-465d-b8b0-4cafec2e9d98.png), and original [see the link here](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10)

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @Marcraphael12** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- visiting this website for more info about the mobile view [see here](https://www.browserstack.com/guide/view-mobile-version-of-website-on-chrome#:~:text=%20Listed%20below%20are%20the%20steps%20to%20view,the%20mobile%20view%20of%20the%20website.%20More%20)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -79,6 +79,139 @@ <h2>Software Developer</h2>
...
+						</ul>
+
+						<button type="button" class="see-project">
+							see project
+						</button>
+					</div>
+				</li>
+		  </ul>
+		</section>
 	</main>
```
[the remark this answers is not in the document]
Comment (Marcraphael12) [reply index.html:215]: Thank you for your comment, but the task is not to create a footer, for now, this is for the next milestone. Also for this template, there is no about section, and the skills list is side the header!

**Review by SpaYco — COMMENTED** [review]
Hello 👋,

_This is an additional review as a result of the second opinion request._

You're right, the project doesn't need the footer for now, also, even though the alignment of the project isn't right, it's not part of this project, however, I highly recommend fixing it (right now, every size is fixed, instead, try making it flexible)

As this was the only reason that your project was not approved it will be approved as soon as possible. It may take few hours, but you should see the status change in your dashboard soon!

Keep rocking 🚀!

**Pull request merged**